tests/css_import/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -63,6 +63,16 @@ $dir: 'fonts';
 @import 'rounded-corners', 'text-shadow';
 $family: unquote("Droid+Sans");
 @import url("http://fonts.googleapis.com/css?family=#{$family}");
+@import url( "foo.css");
+@import url("foo.css" );
+@import url( "foo.css" );
+@import url(  "foo.css"  );
+@import url("foo bar baz.css");
+@import url(  "foo bar baz.css"  );
+@import url("foo  bar  baz.css");
```

Comment:
There are no tests for the case insensitivity?

Reply:
@lydell right now media parsed don't works with `URL` case, i think better fix this in parser then add tests and fixes